### ISSUE TITLE
The current Outlier Removal only runs on clean dataset

### ISSUE THREAD

**Issue by aniqueakhtar** [mrakotosaon/pointcleannet#6]
The current code provided to evaluate the PointCleanNet runs on the validationset.txt files. The same is the case with training. The training is happening on the clean dataset.
All of them are already clean. To evaluate, the PointCleanNet should run on files that have outliers. Not sure which outlier you are using for your pre-trained model. "outliers_uniform" or "outliers_gaussian"

**Comment by aniqueakhtar** [comment]
Furthermore. The Outlier removal's training set is totally different from the validation set.
How does that even work? Which percentage of outliers are used for training?
Which one is used for validation?

**Comment by mrakotosaon** [comment]
Hi @aniqueakhtar, 

- Thank you for your message. Indeed in the current version of the validationset.txt only noisy shapes are listed. Please feel free to modify the file with the shapes names you wish to test with from the test dataset. In our experiments we evaluated on all the shapes from the pointCleanNetOutliersTestSet directory. Including shapes without outliers as we want to measure the robustness of the method.

- For the case of **training** , in train_pcpnet.py we refer to the trainingset.txt and validationset.txt from the directory pointCleanNetOutliersTrainingSet. validationset.txt here is different than the one from the testset. In train_pcpnet.py, I see that we are using both files in row 215 to build the datasets. Is there a problem in the code I missed here?

- To pretrain the model, we used the shapes currently listed in training_set.txt and the shapes in validation_set.txt for validation (from pointCleanNetOutliersTrainingSet directory). Please see section 6.1 of our paper for more details on the dataset.

**Comment by aniqueakhtar** [comment]
I believe it all makes sense to me now. Only the evaluation is happening on the clean set. The training has a different validationset.txt in the different folder that it runs through.
Thank you!
